portal/src/main/java/com/liferay/faces/portal/render/internal/PortalTagRendererCompat.java

```diff
@@ -28,8 +28,12 @@
  */
 public abstract class PortalTagRendererCompat extends Renderer {
 
+	// Public Constants
+	public static final String AUI_FORM_USE_NAMESPACE = "aui:form:useNamespace";
+	public static final String AUI_FORM_PORTLET_NAMESPACE = "aui:form:portletNamespace";
```

Comment:
@juangon Do these need to be public constants since they are only used by PortalTagRenderer?